lib/active_model/serializer/caching.rb

```diff
@@ -88,12 +89,74 @@ def fragmented(serializer)
         # @todo require less code comments. See
         # https://github.com/rails-api/active_model_serializers/pull/1249#issuecomment-146567837
         def cache(options = {})
-          self._cache = ActiveModelSerializers.config.cache_store if ActiveModelSerializers.config.perform_caching
+          self._cache =
+            options.delete(:cache_store) ||
```

Comment:
I don't think there is a test for "Using the `cache_store` options" e.g.

```ruby
class PostSerializer < ActiveModel::Serializer
  cache cache_store: some_cache_store
end
```

Correct me if I'm wrong.